### PR TITLE
Add missing bundles for camel-amqp

### PR DESCRIPTION
**Pull request by lopushen**
Fixes https://jira.talendforge.org/browse/TPRUN-8038

### CODE REVIEW

**Comment by lopushen** [comment]
Tested with
```
<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
           xsi:schemaLocation="
             http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0 http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
             http://www.osgi.org/xmlns/blueprint/v1.0.0 https://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
             http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">

    <bean id="amqpConnectionDetails" class="org.apache.camel.component.amqp.AMQPConnectionDetails">
        <argument value="amqp://localhost:5672"/>
    </bean>
    

    <camelContext id="camelContext" xmlns="http://camel.apache.org/schema/blueprint">
        <route id="amqpRoute">
            <from uri="timer://testTimer?fixedRate=true&amp;period=5000"/>
            <setBody>
                <constant>Hello Camel</constant>
            </setBody>
            <to uri="amqp:queue:yourQueueName"/>
            <log message="Message sent to AMQP: ${body}"/>
            <to uri="amqp:queue:yourQueueName"/>
            <log message="Message received from AMQP: ${body}"/>
        </route>

    </camelContext>

</blueprint>
```
Added missing features

**Comment by essobedo** [comment]
Any reason why you only test the producers? I would expect a second route that consume the messages that you created in the first route

**Comment by lopushen** [comment]
> Any reason why you only test the producers? I would expect a second route that consumes the messages that you create in the first route

you're right, no reason, should be tested - added a consumer and missing bundles for consumer (it was missing 2 bundles)

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -294,6 +294,8 @@
         <bundle dependency='true'>mvn:org.apache.qpid/qpid-jms-client/${qpid-jms-client-version}</bundle>
         <bundle dependency='true'>mvn:org.apache.qpid/proton-j/${qpid-proton-j-version}</bundle>
         <bundle dependency='true'>mvn:io.netty/netty-codec-http/${netty-version}</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-transport-classes-epoll/${netty-version}</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:297]: when consuming the message encountered
```
18:09:12.714 DEBUG [Camel (camel-3) thread #10 - JmsConsumer[yourQueueName]] Unable to check for Epoll support due to missing class definition
java.lang.NoClassDefFoundError: io/netty/channel/epoll/Epoll
	at org.apache.qpid.jms.transports.netty.EpollSupport.isAvailable(EpollSupport.java:37) [!/:?]
	at org.apache.qpid.jms.transports.netty.EventLoopType.valueOf(EventLoopType.java:78) [!/:?]
	at org.apache.qpid.jms.transports.netty.NettyTcpTransport.connect(NettyTcpTransport.java:138) [!/:?]
	at org.apache.qpid.jms.provider.amqp.AmqpProvider.connect(AmqpProvider.java:231) [!/:?]
	at org.apache.qpid.jms.JmsConnection.connect(JmsConnection.java:162) [!/:?]
	at org.apache.qpid.jms.JmsConnectionFactory.createConnection(JmsConnectionFactory.java:214) [!/:?]
	at org.apache.qpid.jms.JmsConnectionFactory.createConnection(JmsConnectionFactory.java:201) [!/:?]
	at org.springframework.jms.support.JmsAccessor.createConnection(JmsAccessor.java:211) [!/:6.1.5]
	at org.springframework.jms.listener.AbstractJmsListeningContainer.createSharedConnection(AbstractJmsListeningContainer.java:405) [!/:6.1.5]
	at org.springframework.jms.listener.AbstractJmsListeningContainer.refreshSharedConnection(AbstractJmsListeningContainer.java:390) [!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer.refreshConnectionUntilSuccessful(DefaultMessageListenerContainer.java:1004) [!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer.recoverAfterListenerSetupFailure(DefaultMessageListenerContainer.java:978) [!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1166) [!/:6.1.5]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.ClassNotFoundException: io.netty.channel.epoll.Epoll not found by org.apache.qpid.jms.client [249]
	... 16 more
```

The message was not consumed

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -294,6 +294,8 @@
...
+        <bundle dependency='true'>mvn:io.micrometer/micrometer-observation/${micrometer-version}</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:298]: when consuming the message encountered
```
18:09:17.686 WARN [Camel (camel-3) thread #8 - JmsConsumer[yourQueueName]] Setup of JMS message listener invoker failed for destination 'yourQueueName' - trying to recover. Cause: io/micrometer/observation/Observation
java.lang.NoClassDefFoundError: io/micrometer/observation/Observation
	at org.springframework.jms.listener.AbstractMessageListenerContainer.createObservation(AbstractMessageListenerContainer.java:693) ~[!/:6.1.5]
	at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.doReceiveAndExecute(AbstractPollingMessageListenerContainer.java:320) ~[!/:6.1.5]
	at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.receiveAndExecute(AbstractPollingMessageListenerContainer.java:270) ~[!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.invokeListener(DefaultMessageListenerContainer.java:1257) ~[!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.executeOngoingLoop(DefaultMessageListenerContainer.java:1247) ~[!/:6.1.5]
	at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1140) [!/:6.1.5]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) [?:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.ClassNotFoundException: io.micrometer.observation.Observation not found by wrap_file__Users_ylopushen_.m2_repository_org_springframework_spring-jms_6.1.5_spring-jms-6.1.5.jar_Export-Package_org.springframework.__version_6.1.5 [251]
```
The message was not consumed

**Comment by lopushen** [comment]
The fix was re-visited and implemented in https://github.com/apache/camel-karaf/pull/272, closed this one

**Pull request closed without merging**